app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/CommonTriageQuestionsController.scala

```diff
@@ -647,7 +654,10 @@ class CommonTriageQuestionsController @Inject() (
               if (alreadySentSA)
                 Redirect(routes.CommonTriageQuestionsController.selfAssessmentAlreadySubmitted())
               else if (alreadySentSelfAssessment.contains(alreadySentSA))
-                Redirect(routes.MultipleDisposalsTriageController.checkYourAnswers())
+                triageAnswersFomState(state).isLeft match {
+                  case true  => Redirect(routes.MultipleDisposalsTriageController.checkYourAnswers())
+                  case false => Redirect(routes.SingleDisposalsTriageController.checkYourAnswers())
+                }
```

Comment:
Yes right, it was missing. 
Please use this instead of that logic `Redirect(redirectToCheckYourAnswers(state))`
Already using bottom of that function. probably missed at top